pep-0696.rst

```diff
@@ -345,7 +432,7 @@ with non-defaults can be checked at compile time.
 Rejected Alternatives
 ---------------------
 
-Allowing the ``TypeVarLike``\ s defaults to be passed to ``type.__new__``'s ``**kwargs``
+Allowing The ``TypeVarLike``\ s Defaults To Be Passed To ``type.__new__``'s ``**kwargs``
 ''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''
```

Comment:
Whether the first "to" is capitalized depends on whether you follow AP that says yes for to in infinitives, or most of the other style guides (APA, MLA, Chicago, etc) that say no. I generally follow AP by default, but it is somewhat of an outlier here so its really up to you

```suggestion
Allowing the ``TypeVarLike``\s Defaults to Be Passed to ``type.__new__``'s ``**kwargs``
'''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''
```

or

```suggestion
Allowing the ``TypeVarLike``\s Defaults To Be Passed to ``type.__new__``'s ``**kwargs``
'''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''
```